sdk/include/opentelemetry/sdk/metrics/controller.h

```diff
@@ -59,9 +59,8 @@ class PushController
    */
   bool start()
   {
-    if (!active_.load())
+    if (!active_.exchange(true))
```

Comment:
Please fix spelling above --- `currecntly` in `isActive` comment. It may be worthwhile mentioning that `isActive` method itself returns the `best guess` state. It may not be the actual state on start or on stop. Not sure under what circumstances you want to use this method, but it may present a challenge. Atomic boolean in this context is creating an illusion of thread safety.